package.json

```diff
@@ -69,7 +69,7 @@
     "@angular/platform-browser": "^7.2.13",
     "@angular/platform-browser-dynamic": "^7.2.13",
     "@angular/router": "^7.2.13",
-    "@hmcts/ccd-case-ui-toolkit": "4.1.0-rc.13",
+    "@hmcts/ccd-case-ui-toolkit": "v4.0.0-strategic-activity-tracker-gamma",
```

Comment:
(Nitpick)
Thought we weren't using "v" anymore.

Reply:
This is a tag-release version for testing purposes and will change once it's been QAed. I've always stuck a "v" at the front of my tags... am I not supposed to? 😬🤷‍♂️

Reply:
According to https://tools.hmcts.net/confluence/display/EUI/Release+process+effective+sprint+22.1 you don't need the v at the front. Anyhow, not a showstopper!

Reply:
No "v" on the new one. 😁